src/v/utils/BUILD

```diff
@@ -1,5 +1,19 @@
 load("//bazel:build.bzl", "redpanda_cc_library")
 
+package(default_visibility = ["//visibility:public"])
```

Comment:
This is the way! Test folders should have testonly = True set if there are cc_library targets